src/NUnitFramework/framework/Api/NUnitTestAssemblyRunner.cs

```diff
@@ -232,11 +232,7 @@ public void RunAsync(ITestListener listener, ITestFilter filter)
         /// <returns>True if the run completed, otherwise false</returns>
         public bool WaitForCompletion(int timeout)
         {
-#if !PORTABLE && !NETSTANDARD1_6
-            return _runComplete.WaitOne(timeout, false);
-#else
             return _runComplete.WaitOne(timeout);
-#endif
         }
```

Comment:
I think the `.WaitOne(int, bool)` overload was missing from some platforms we no longer support.